README.md

```diff
@@ -1 +1,107 @@
-# peer-did-python
\ No newline at end of file
+# Peer DID python API
+
+## Information
+
+This is a work-in-progress implementation of the Peer DID specification:
```

Comment:
I would remove the `work-in-progress` statement. We are considering static support is enough for many cases and can be considered as a valuable MVP.